Sparkle/SUTouchBarButtonGroup.m

```diff
@@ -0,0 +1,60 @@
+//
+//  SUTouchBarButtonGroup.m
+//  Sparkle
+//
+//  Created by Yuxin Wang on 05/01/2017.
+//  Copyright © 2017 Sparkle Project. All rights reserved.
+//
+
+#import "SUTouchBarButtonGroup.h"
+
+@implementation SUTouchBarButtonGroup
+
+@synthesize buttons = _buttons;
+
+- (instancetype)initByReferencingButtons:(NSArray<NSButton *> *)buttons
```

Comment:
I'm very sorry than I still can't accept your suggestion now. Could you please merge this pull request and change the API later if you insist?
I am fine with the API as you suggest, but I don't want to change it in my commit because it is against my will. I hope you can change it in your commit, to keep the fact that we have different designs.